ui/apps/platform/cypress/integration/vulnmanagement/dashboard.test.js

```diff
@@ -64,14 +64,13 @@ describe('Vuln Management Dashboard Page', () => {
 
     it('should show same number of images between the tile and the images list', () => {
         visitVulnerabilityManagementDashboard();
-        cy.get(selectors.tileLinks)
-            .eq(2)
-
+        cy.get(selectors.tileLinks, { timeout: 8000 })
+            .eq(3)
```

Comment:
Not sure why this was set to 2 🤔 Maybe the `Images` tile was in that position before, but not it should be index 3